plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/handler/ScaleDeploymentHandler.java

```diff
@@ -55,6 +58,13 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
 			return null;
 		}
 		IServiceWrapper deployment = getSelectedElement(event, IServiceWrapper.class);
+		if(deployment == null) {
+			IResourceWrapper<?, IOpenshiftUIElement<?,?>> wrapper = getSelectedElement(event, IResourceWrapper.class);
+			if(wrapper != null && wrapper.getWrapped() instanceof IPod
+					&& !ResourceUtils.isBuildPod((IPod)wrapper.getWrapped())) {
+				deployment = (IServiceWrapper)wrapper.getParent();
```

Comment:
how can we be sure that the parent of the wrapped resource is a IServiceWrapper?